### PR TITLE
correction README.md 

### PR DESCRIPTION
**Pull request by saumyaagrawal2001**
## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you have any questions, queries or comments, [RocketChat] is the place to fin
 
 ## Besu Users
 
-Users wanting to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
+Users want to install Besu should install the Besu binary by following [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).
```
Comment (shemnon) [inline README.md:32]: Should it be "who want" instead?  @MadelineMurray

Comment (macfarla) [reply README.md:32]: Maybe it should just be "To install the Besu binary, follow these instruction ..."

Comment (MadelineMurray) [reply README.md:32]: +1 to @macfarla's suggestion

Comment (timbeiko) [inline README.md:33]:
```suggestion
To install the Besu binary, follow [these instructions](https://besu.hyperledger.org/HowTo/Get-Started/Install-Binaries/).    
```

Comment (timbeiko) [reply README.md:33]: @saumyaagrawal2001 if you commit my suggestion above, and add a signoff in the commit message, we should be able to merge this.

**Comment by timbeiko** [comment]
@saumyaagrawal2001 to commit this, you will need to signoff the commits. See https://github.com/hyperledger/besu/pull/1460/checks?check_run_id=1344322629

**Comment by timbeiko** [comment]
Closed in favor of https://github.com/hyperledger/besu/pull/1574

**Pull request closed without merging**